python/scrutiny/server/protocol/protocol.py

```diff
@@ -50,6 +53,12 @@ def __init__(self, version_major=1, version_minor=0, address_size=32):
     def set_address_size(self, address_size):
```

Comment:
This is confusing. The set method name should look like the get method name.

```suggestion
    def set_address_size_bytes(self, address_size):
```

Reply:
either that or rename `get_address_size_bytes` to `get_address_size` and adopt/follow that bytes are implicit what it comes to size.

Reply:
Will fix